Assets/Scripts/Game/UserInterfaceWindows/DaggerfallPauseOptionsWindow.cs

```diff
@@ -293,6 +293,15 @@ private void ControlsButton_OnMouseClick(BaseScreenComponent sender, Vector2 pos
         private void FullScreenButton_OnMouseClick(BaseScreenComponent sender, Vector2 position)
         {
             fullScreenTick.Enabled = DaggerfallUnity.Settings.Fullscreen = Screen.fullScreen = !Screen.fullScreen;
+
+            if (DaggerfallUnity.Settings.Fullscreen && DaggerfallUnity.Settings.ExclusiveFullscreen)
```

Comment:
Just FYI, I'll merge this as-is but the Fullscreen button in DF will soon be reverted to toggle large HUD on and off. It had a different meaning in classic than the modern fullscreen/windowed.